packages/framer-motion/src/render/html/utils/build-styles.ts

```diff
@@ -59,31 +56,38 @@ export function buildHTMLStyles(
             if (!transformIsNone) continue
 
             // Otherwise check to see if this is a default transform
-            if (value !== (valueType.default ?? 0)) transformIsNone = false
-        } else if (isTransformOriginProp(key)) {
-            transformOrigin[key] = valueAsType
-
+            if (value !== (valueType.default || 0)) transformIsNone = false
+        } else if (key.startsWith("origin")) {
             // If this is a transform origin, flag and enable further transform-origin processing
             hasTransformOrigin = true
+
+            transformOrigin[key] = valueAsType
         } else {
             style[key] = valueAsType
         }
     }
 
-    if (hasTransform) {
+    if (hasTransform || transformTemplate) {
```

Comment:
Does `buildTransform` output the same result as `transformTemplate({}, "")`? Also why do we need the `transformTemplate` argument? It doesn't seem to be used at all in `buildTransform` 👀

Reply:
It is, near the bottom (line 53)

But yeah this would basically be the same thing as passing `buildTransform` an empty object

Reply:
If the state we pass in here does have a non-empty `transfrom`, it wouldn't be the same no? 🤔 
```
if (transformTemplate) {
        transformString = transformTemplate(
            transform,
            transformIsDefault ? "" : transformString
        )
    } 
```

Reply:
Yeah but the previous situation was

- If we have a transform, `buildTransform`
  - If we have a `transformTemplate`, use that
- If we don't have a transform, but we do have a `transformTemplate`, use that

Whereas now it's
- `buildTransform`
  - If we have a `transformTemplate`, use that
  
Which are the same.

Reply:
Ah I see, I thought the arguments we passed in were a bit different, but I missed that if we don't have a transform, `transformIsDefault` will be true, so we will be passing in an empty string anyway. Thanks for explaining!